README.md

```diff
@@ -1,79 +1,230 @@
+### 4. Go to the [Application Installation](#application-installation) section, and continue following the steps.
 
-7. If you cannot successfully connect to the database endpoint, install [mongosh](https://www.mongodb.com/docs/mongodb-shell/install/). After installing, run the following command in the root directory:
+<hr>
 
-```properties
-mongosh
-```
+## Windows Setup
+
+### 1. Install Git
+- Follow the steps to install [Git](https://git-scm.com/downloads)
+
+### 2. NPM / Node.js
+- Follow the steps to install version 22.x.x of [Node.js](https://nodejs.org/en/download/package-manager), which comes prepackaged with NPM, a dependency installer for Javascript
 
-Then, in the Mongo Shell, run these commands:
+### 3. Go to the [Application Installation](#application-installation) section, and continue following the steps.
 
+<hr>
+
+## Application Installation
+- This section defines applications you will need to run / develop the project.
+
+### 1. Install Visual Studio Code
+- Follow the steps to install [Visual Studio Code](https://code.visualstudio.com/download)
+- Launch Visual Studio Code.
+- Navigate to the 'Extensions' tab, and download the following:
+  - Jest
+  - Prettier
+  - Live Share
+
+### 2. Install Github Desktop
+- Follow the steps to install [Github Desktop](https://desktop.github.com/download/)
+- Launch Github Desktop, and go to 'Settings'.
+- Click the 'Git' tab, and connect to your Github account to the application.
+
+### 3. Install Docker Desktop
+- Follow the steps to install [Docker Desktop](https://www.docker.com/products/docker-desktop/)
+- Launch Docker Desktop, no need to sign in or create an account.
+
+### 4. Install MongoDB Compass
+- Follow the steps to install [MongoDB Compass](https://www.mongodb.com/products/tools/compass)
+- Launch MongoDB Compass.
+
+### 5. You're done installing all the necessary software, congrats!
+- To build and run the project locally, refer to the [Running the Project](#running-the-project) section.
+
+<hr>
+
+# Running the Project
+- This section defines how to start up the project, and run it on your local machine.
+
+### 1. Clone the repository to your computer
+- Navigate to this page, and click the "Code" button. Click the "HTTPS" tab, and copy the repository URL.
+- Open Github Desktop, and click the "Add" option, and then "Clone Repository". Select the "URL" option, and paste the URL from Github.
+
+### 2. Open the Project in Visual Studio Code
+- Open the project directory in Visual Studio Code by either clicking "Open in Visual Studio Code" on Github Desktop, or launching Visual Studio Code, clicking "Open Folder", and navigating to the directory in which you cloned the project.
+- Add the ```.env``` file to the root directory. This will define our important secret keys we use for authentication, and initialization of the project.
```

Comment:
maybe just mention who to contact / how to get the actual .env values so no one forgets that part